_projects/fmm_project.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: page_project
+title: Scalability Enhancements to FMM for Molecular Dynamics Simulations
+subtitle:
+date: 01/01/2016
+update:
+navbar: Projects
+subnavbar:
+footer: true
+logo:
+project_url:
+short_desc:
+---
+<!---
+Questions/requests:
+
+* we need a naming convention for the project_template...
+
+* what is "update" and "subnavbar"?
```

Comment:
I thought, I had described that in the wiki. Unfortunately, I didn't. Or it got lost. Will fix that.